### ISSUE TITLE
Setting size of atoms

### ISSUE THREAD

**Issue by cnrd** [cnrd/IDE15-16#17]
I just tried setting the size using the 'radius' attribute. but x3dom is just spewing errors.

This works:

    atomScene.append('transform').attr('translation','-52.822 -1.611 23.137')
		      .append('shape').append('sphere').attr('class', atoms').attr('radius', '0.5')
		      .select(function(){
						return this.parentNode;
			 		})
		      .append('appearance').append('material').attr('diffuseColor','red');

This does not work:

    	atomScene.selectAll('sphere')
			 .data(data)
			 .enter()
			 .append('transform').attr('translation', function(d){
			 	// Setting x, y, z coordinates
			 	// No reason to do scaling, we can just move the viewport
			 	return d.x + ' ' + d.y + ' ' + d.z;
			 })
			 .append('shape').append('sphere').attr('class','atoms').attr('radius', '0.5')
			 .select(function(){
			 	// Apperance should be inside the 'shape node'
			 	return this.parentNode;
			 })
			 .append('appearance').append('material').attr('diffuseColor', function(d){
			 	// Setting the color of atoms
			 	return colorAtom(d.element);
			 });

Even when the actual html code before x3dom translation is the exact same...

**Comment by koefoed** [comment]
In the first piece of code, line 2: count the number of ' (single quotation mark) around "atoms".

Hard to tell. Make a branch?

Maybe make the shape into a variable so that you can invoke multiple appends on it? À la:
```
var shape = atomScene.selectAll('sphere')
         .data(data)
         .enter()
         .append('transform').attr('translation', function(d){
            // Setting x, y, z coordinates
            // No reason to do scaling, we can just move the viewport
            return d.x + ' ' + d.y + ' ' + d.z;
         })
         .append('shape');

shape.append('sphere').attr('class','atoms').attr('radius', '0.5');
shape.append('appearance').append('material').attr('diffuseColor', function(d){
            // Setting the color of atoms
            return colorAtom(d.element);
         });
```
(without the .select... return ...parent stuff)
I don't know if this makes a difference.

**Comment by cnrd** [comment]
Yep found the missing ' but that was just an error when copying.

I added a scale attribute, like this:

    	atomScene.selectAll('sphere')
			 .data(data)
			 .enter()
			 .append('transform').attr('scale', '0.5 0.5 0.5').attr('translation', function(d){
			 	// Setting x, y, z coordinates
			 	// No reason to do scaling, we can just move the viewport
			 	return d.x + ' ' + d.y + ' ' + d.z;
			 })
			 .append('shape').append('sphere').attr('class','atoms')
			 .select(function(){
			 	// Apperance should be inside the 'shape node'
			 	return this.parentNode;
			 })
			 .append('appearance').append('material').attr('diffuseColor', function(d){
			 	// Setting the color of atoms
			 	return colorAtom(d.element);
			 });

Not sure if it will have any later implications.
